packages/flutter/test/rendering/slivers_test.dart

```diff
@@ -13,8 +13,39 @@ void main() {
     final RenderViewport root = new RenderViewport(
       offset: new ViewportOffset.zero(),
     );
+    // TODO(jacobr): remove extra trailing line break.
+    expect(root, isNot(hasAGoodToStringDeep));
```

Comment:
We want it to be correct. The TODO on the previous line and the test expectation
will be flipped when I next land a cl that removes the extra line.